portfolio-project-react/src/components/WorkExperience/WorkExperience.jsx

```diff
@@ -23,7 +23,10 @@ import WorkExpItem from "../WorkExperienceItem/WorkExperienceItem";
 import moment from "moment";
 import styles from "./WorkExperience.module.css";
 
-const WorkExperience = ({ isEditModeEnabled }) => {
+const WorkExperience = ({
+  isEditModeEnabled,
+  onUpdateTotalExperience,
```

Comment:
instead of passing `onUpdateTotalExperience` as a callback and updating the parent's state from the child component. its better to "lift state up" to the parent component, do the calculations there and pass the needed information from parent to child components.

This concept in react is called "Lifting state up" and is a good practice to follow.
You can read more about it here https://react.dev/learn/sharing-state-between-components or research on your own and learn about this concept.